### ISSUE TITLE
Use w3c/distributed-tracing propagation format as default for HTTP

### ISSUE THREAD

**Issue by rakyll** [census-instrumentation/opencensus-go#921]
OpenCensus Go should plan switching from B3 to https://github.com/w3c/distributed-tracing as the default HTTP propagation format as we are trying to unify the context propagation around this proposal.

I suggest us to cut a new breaking release and document how to switch back to B3 (if necessary for backwards compatibility) on the release notes.

/cc @ramonza

**Comment by ramonza** [comment]
I don't think we should silently change behavior without requiring any code change. I would prefer to keep the current default and just change all the examples. Most users copy and paste from examples anyway, so this should ultimately have the same effect.

If we want to push harder for people to move, we could deprecate ochttp.Transport and replace with something else (ochttp.NewRoundTripper() maybe?) that has the new behavior.

Also think the default should be documented in the specs repo.

**Comment by rakyll** [comment]
We can change the default behavior if we mark it as a breaking change. It is not correct if we keep it as B3 as other libraries are going to use the new standard. I dislike the idea of them relying on examples for the desired default behavior. I am also not a big fan of introducing a non-ideal API.

**Comment by rakyll** [comment]
Also forgot to say, there is a way to fallback to the old behavior from the new version. It is not disabling anyone on the upgrade path.

**Comment by ramonza** [comment]
What do you mean by "mark it as a breaking change"? You mean a major version change i.e. we move to 1.X.Y?

**Comment by ramonza** [comment]
This is my proposal: https://github.com/census-instrumentation/opencensus-go/pull/930

**Comment by rakyll** [comment]
We don't cut major versions yet, in this case we can just document in the release notes the change and offer the workaround.
